corehq/messaging/smsbackends/tropo/views.py

```diff
@@ -12,7 +12,6 @@
 from corehq.apps.sms.util import strip_plus
 
 
-@csrf_exempt
 def sms_in(request, backend_id=None):
```

Comment:
Is this view still used? If not, why not remove it too?

Trying to understand the removal of `@csrf_exempt`. Would be helpful if you could explain why/how that was necessary/possible if this is still being used.

Reply:
this used to be the view that was being invoked so it needed the csrf_exempt decorator. but now this is just a function that's called by the class-based view (which has its own csrf_exempt decorator). so basically we need to the keep the function but not the decorator.